quickstart/2-browser-parallel-computation/aqua/getting-started.aqua

```diff
@@ -0,0 +1,58 @@
+import "@fluencelabs/aqua-lib/builtin.aqua"
+import Subnet, Worker from "@fluencelabs/aqua-lib/subnet.aqua"
+
+use "deals.aqua"
+import "services.aqua"
```

Comment:
why `use` in one, and `import` in the other?

Reply:
also let's not promote nameless `import`. `import X from Y` is much better than `import Y`.

Reply:
```
4 use "services.aqua"
      ^^^^^^^^^^^^^^^
                 Used module has no `module` header. Please add `module` header or use ... as ModuleName, or switch to import

```
use import wont work there

Reply:
I'd do
```
-- 'use' clause brings Deal service into the scope
use deals.aqua

-- 'import from' allows for fine-grained import, makes code easier to follow
import services.aqua
```

And maybe `services.aqua` should provide `module` clause? If not, why not? cc @shamsartem

Reply:
Is it Deal service though? I always visited l considered this to be just named import syntax, where Deal is the name of the module

You mean I should add 

aqua Services declares *

To the services.aqua?

Probably wouldn't hurt, import statements still work with this and import statements in this case are more convenient to use then named imports (with use)